api-reference/beta/api/application_list_extensionproperties.md

```diff
@@ -1,8 +1,8 @@
 # List extensionProperties
 
-Retrieve a list of extensionproperty objects.
+Retrieve a list of extensionProperty objects.
```

Comment:
Not sure if we want to publish this information.  If we do, can we make clear that this is for Azure AD Directory Schema Extensions (i.e. this is the "legacy" version of schema extensions).